src/fe-text/gui-readline.c

```diff
@@ -1278,7 +1279,7 @@ void gui_readline_init(void)
 
         /* line transmitting */
 	key_bind("send_line", "Execute the input line", "return", NULL, (SIGNAL_FUNC) key_send_line);
-	key_bind("word_completion_backward", "", NULL, NULL, (SIGNAL_FUNC) key_word_completion_backward);
+	key_bind("word_completion_backward", "Completes the previous word", "shifttab", NULL, (SIGNAL_FUNC) key_word_completion_backward);
```

Comment:
This is confusing, it was the first thing I read in this PR and what I interpreted was that, given `a b<Tab>` it completes based on `a` instead of `b`

Reply:
Hard to come up with a good description.. maybe something like "Choose previous completion suggestion" ?

Reply:
Ok done :P Dunno if "suggestion" word if redundant tho?